xicam/gui/bluesky/search.py

```diff
@@ -112,7 +112,7 @@ def __init__(self, catalog):
         self.new_results_catalog.connect(self.start_show_results)
 
         class ReloadThread(QThread):
-            def run(self):
+            def run(thread):
```

Comment:
This comes up as as warning in in my IDE...should be self.

Reply:
Moved these classes outside